stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/RetrofitCallAdapterFactory.kt

```diff
@@ -24,21 +28,31 @@ internal class RetrofitCallAdapterFactory(
             throw IllegalArgumentException("Call return type must be parameterized as Call<Foo>")
         }
         val responseType: Type = getParameterUpperBound(0, returnType)
-        return RetrofitCallAdapter<Any>(responseType, chatParser)
+        return RetrofitCallAdapter<Any>(responseType, chatParser, callbackExecutor)
     }
 
     companion object {
-        fun create(chatParser: ChatParser) = RetrofitCallAdapterFactory(chatParser)
+        val mainThreadExecutor by lazy {
```

Comment:
We probably don't need `lazy` here, we'll instantiate this immediately when our SDK is initialized anyway

Reply:
`Lopper.getMainLooper()` is Android SDK stuff that is not accesible in our tests. Because we are using this static field, some of our tests are using it and they crash if they try to get the Main Looper

Reply:
Ah, I see, we're mocking the Retrofit Api interface itself, and this class is being used with its real implementation in tests. Makes sense then 👍

Reply:
In the future maybe we should try to wrap this Looper or think into another "default argument implementation" into our `RetroSuccess` class